src/modules/profile/pages/profileSettings/views/ProfileSettingsWallet/ResetAccountPasswordSection.tsx

```
@@ -0,0 +1,86 @@
+import { t } from 'i18next'
+
+import {
+  RecoverPasswordForm,
+  useRecoverPasswordForm,
+} from '@/modules/project/forms/accountPassword/components/RecoverPasswordForm.tsx'
+import { Modal } from '@/shared/components/layouts/Modal.tsx'
+import { GuideUrl } from '@/shared/constants/platform/url.ts'
+import { Body } from '@/shared/components/typography/Body.tsx'
+import { H2 } from '@/shared/components/typography/Heading.tsx'
+import { Feedback, FeedBackVariant } from '@/shared/molecules/Feedback.tsx'
+import { useNotification } from '@/utils/index.ts'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)